dist/other/iframe-messaging/IframeToFromHostMessageHandler.js

```diff
@@ -1,6 +1,11 @@
+import { Action as ActionEnum, Type as TypeEnum, } from "@mat3ra/esse/dist/js/types";
```

Comment:
This is the issue

Reply:
Yot can not import *.ts in *.js file

Reply:
But this is dist that was generated

Reply:
Probably this thing expects us to convert types.ts to types.js as well